css-grid_place-elements/css/styles.css

```diff
@@ -68,24 +68,38 @@ body {
 
 .element-1 {
   /* Add styles here */
+  grid-column-start: 3;
+  grid-column-end: 5;
+
 }
 
 .element-2 {
   /* Add styles here */
+  grid-column-start: 5; 
+  grid-column-end: 11;
 }
 
 .element-5 {
   /* Add styles here */
+  grid-column-start: 3;
+  grid-column-end: 5;
+  grid-row: span 2;
```

Comment:
That's a nice idea, better than mine, I explicitly defined the start and end, your solution is cleaner.